### PR TITLE
Expose session.config

### PR DESCRIPTION
**Pull request by marcusoffesson**
With this change, the global configuration object can be accessed through `session.config`.

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -211,6 +212,10 @@ to modify the result or reject the promise chain before it is sent to mixins.
 
 You retrieve a session by calling [`enigma.create(config)`](#enigmacreateconfig).
 
+### `session.config`
+
+A reference to this session's configuration object (including default values for optional parameters).
```
Comment (peol) [inline docs/api.md]: this=a to keep the grammar consistent (I think we refer to "a session" in the other API sections)?

Comment (peol) [reply docs/api.md]: Perhaps link the configuration text to the configuration section?

File: test/integration/suspend-resume.spec.js [hunk 2]
```diff
@@ -69,15 +69,15 @@ describe('QIX Suspend/Resume', () => {
...
+      .then(() => session.suspend())
       .then(() => expect(suspended.calledOnce).to.equal(true))
       .then(() => global.session.resume())
       .then(() => app.getAppProperties())
       // verify that we have reconnected to a fresh app, since we never saved
       // this property it shouldn't exist in a new one:
       .then(props => expect(props.test).to.equal(undefined))
       .then(() => global.deleteApp(app.id))
-      .then(() => global.session.close())
+      .finally(() => session.close())
```
Comment (peol) [inline test/integration/suspend-resume.spec.js]: `finally()` is not part of the ES6 spec (yet, AFAIK there are discussions but nothing decided), we may want to note that we're using the bluebird superset here.

Comment (marcusoffesson) [reply test/integration/suspend-resume.spec.js]: Good point, I'll remove it here (and create a new issue about removing all bluebird references).

**Pull request merged**